sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/misc.scala

```diff
@@ -189,7 +189,7 @@ object AssertTrue {
 case class CurrentDatabase() extends LeafExpression with Unevaluable {
   override def dataType: DataType = StringType
   override def nullable: Boolean = false
-  override def prettyName: String = "current_database"
+  override def prettyName: String = "current_schema"
```

Comment:
If we change current_database to current_schema, we should rename `CurrentDatabase` as `CurrentSchema`.

How about using `current_database` as a unified name?

Reply:
cc @zhengruifeng @HyukjinKwon

Reply:
`CurrentDatabase` was there since 1.6.0 and seems safer to not rename it now. The expression class name is not user-facing anyway.

Reply:
AFAIK schema is a more standard name to reference to a namespace that keeps database objects.

Reply:
So, how about using `current_database` as a unified name? it is consistent with class name.

Reply:
Sorry I'm a bit confused. This PR tries to use the name `schema` more widely as it's more SQL standard. Are you saying no to this PR and want to stick with `database`?

Reply:
I know this PR want to use the name `schema` here, it's okay. However, it caused inconsistent between the class name of the `CurrentDatabase` and its `prettyName`. Is this a problem? If not, then it's okay.

Reply:
It's fine, `TruncTimestamp` overrides `prettyName` as `date_trunc`